contentctl/objects/drilldown.py

```diff
@@ -0,0 +1,6 @@
+from pydantic import BaseModel, Field
+class Drilldown(BaseModel):
+    name: str = Field(...,min_length=5)
+    search: str = Field(...,description= "The drilldown search. The drilldown MUST begin with the | character followed by a space.", pattern=r"^\|\s+.*")
```

Comment:
Just checked our savedsearches.conf and can confirm this isn't the case, a drilldown can be any search string, such as a 'raw' search.

We use drilldowns on all of our searches and simply copy the correlation search over to the drilldown as a way to quickly jump to the search results. Very useful for searches with a join or that have a tendency to show different results (or none in the case of a join!) when run over a broader time period.